scripts/install.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env sh
+
+set -e
```

Comment:
Why is `set -e` needed?

Reply:
`set -e` tells the shell to exit if any command returns non-zero. Without it, this script would continue executing even if the download failed:

```sh
curl -O https://cli.download.com/install.sh
ls install.sh
```

Reply:
Makes sense, that is pretty cool!